packages/neso/package.py

```diff
@@ -49,17 +49,19 @@ class Neso(CMakePackage):
     variant(
         "coverage", default=False, description="Enable coverage reporting for GCC/Clang"
     )
+    variant(
+        "nvcxx", default=False, description="Enable compilation using nvcxx"
+    )
 
     # Some SYCL packages require a specific run-time environment to be set
     depends_on("sycl", type=("build", "link"))
     depends_on("intel-oneapi-dpl", when="^dpcpp", type="link")
     depends_on("fftw-api", type="link")
-    depends_on("nektar@5.2.0-2022-09-03+compflow_solver", type="link")
+    depends_on("nektar+compflow_solver", type="link")
     depends_on("cmake@3.14:", type="build")
     depends_on("boost@1.74:", type="test")
     depends_on("googletest+gmock", type="link")
     depends_on("neso-particles")
-    depends_on("mpi", type=("build", "run"))
```

Comment:
Why isn't MPI needed anymore?

Reply:
Ah, I was working off an old commit that was included with NESO when I cloned it (aacfb84), the only changes that git thinks I've made to this file (relative to that old commit) are adding `variant("nvcxx", ...)` and the if statement to append to args if "+nvcxx" is in `self.spec`. I'll revert/update the other changes and test the NESO build again.

Reply:
This should be added back now (see message for commit e2b6e79)